Lesson6.ipynb

```diff
@@ -27,7 +34,686 @@
+    "\n",
+    "print('Applying amax() function with axis = 1:')\n",
+    "print(np.amax(values, axis = 0), '\\n')\n",
+    "\n",
+    "### Percentile\n",
+    "print('Applying percentile() function:')\n",
+    "print (np.percentile(values,50), '\\n')\n",
+    "\n",
+    "print('Applying percentile() function along axis 1:')\n",
+    "print(np.percentile(values,50, axis = 1), '\\n')\n",
+    "\n",
+    "### Median\n",
+    "print('Applying median() function:')\n",
+    "print(np.median(values), \"\\n\") \n",
+    "\n",
+    "print('Applying median() function along axis 0:')\n",
+    "print(np.median(values, axis = 0),'\\n') \n",
+    "\n",
+    "### Mean\n",
+    "print('Applying mean() function:') \n",
+    "print(np.mean(values), '\\n') \n",
+    "\n",
+    "print('Applying mean() function along axis 0:')\n",
+    "print(np.mean(values, axis = 0), '\\n')\n",
+    "\n",
+    "### Average\n",
+    "print('Applying average() function:') \n",
+    "print(np.average(values), '\\n') \n",
+    "\n",
+    "### Standard Deviation\n",
+    "print('Applying std() function:') \n",
+    "print(np.std(values), '\\n') \n",
+    "\n",
+    "### Variance\n",
+    "print('Applying var() function:') \n",
+    "print(np.var(values), '\\n') "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Challenges (Optional)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "p1_daily_steps = [11980, 10437, 17616, 24586, 16136, 13700, 39812, 9195, 12855, 11309, 23606, 11848, 6120, 6254, 8754, 6469, 8849, 9911, 7709, 534, 13465, 7341, 11230, 7878, 11029, 8790, 9006, 21942]\n",
+    "p2_daily_steps = [22935, 13399, 25098, 29581, 26121, 12805, 16073, 15124, 16011, 6198, 10026, 10909, 14468, 4828, 11207, 7133, 14977, 13746, 12267, 9364, 1061, 6075, 11188, 11472, 10150, 13023, 6769, 10165]\n",
+    "p3_daily_steps = [16272, 17231, 20595, 17047, 15216, 42590, 10969, 20687, 19170, 12703, 17192, 12865, 10960, 9105, 16019, 12646, 10042, 13353, 16072, 41673, 13425, 11262, 7801, 6666, 5276, 11353, 5344, 6282]\n",
+    "p4_daily_steps = [10233, 16120, 12897, 24680, 13060, 20489, 10230, 25565, 10029, 12696, 13938, 9475, 5297, 8573, 9857, 15341, 9482, 11649, 5804, 11080, 6245, 7611, 8401, 5596, 6491, 7637, 7610, 9130]\n",
+    "p5_daily_steps = [14126, 14110, 10111, 20440, 21416, 16989, 25371, 21539, 23045, 20043, 20328, 12058, 12004, 3301, 9789, 6671, 7893, 9589, 10459, 5091, 6329, 6784, 6543, 17984, 13588, 11077, 7856, 20897]\n",
+    "\n",
+    "team = { # a dictionary mapping initial to list of daily steps\n",
+    "    'G': p1_daily_steps, \n",
+    "    'I': p2_daily_steps, \n",
+    "    'P': p3_daily_steps, \n",
+    "    'T': p4_daily_steps, \n",
+    "    'D': p5_daily_steps\n",
+    "}\n",
+    "\n",
+    "# Print minimum and maximum no. of steps walked by each person in this format: 'G's (minimum, maximum) steps per day are: (534, 24586)'\n",
```

Comment:
G's maximum is actually 39812.

Reply:
wow, you measured or coded? appreciate such a deep review :)